libr/main/radiff2.c

```diff
@@ -140,14 +140,15 @@ static int cb(RDiff *d, void *user, RDiffOp *op) {
 				printf ("-0x%08"PFMT64x":", op->a_off);
 				int len = op->a_len; // R_MIN (op->a_len, strlen (op->a_buf));
 				for (i = 0; i < len; i++) {
-					printf ("%02x", op->a_buf[i]);
+					printf ("%02x ", op->a_buf[i]);
```

Comment:
dont put spaces between hex numbers. this is an hexpair string

Reply:
Yeah, but I think it's easy to look at if we separate each byte with space.
Especially with long hex string, it's really hard to see.